CRM/Cplabels/Form/Search/Cplabels/Client.php

```diff
@@ -125,18 +125,15 @@ public function buildForm(&$form) {
      */
     $form->assign('elements', array('team', 'diagnosis', 'min_age', 'max_age', 'limit'));
 
-    $form->addRule('team', E::ts('Field "Team" is required'),'required');
-    $form->addRule('diagnosis', E::ts('Field "Diagnosis" is required'),'required');
+    $form->addRule('team', E::ts('Field "Team" is required'), 'required');
+    $form->addRule('diagnosis', E::ts('Field "Diagnosis" is required'), 'required');
   }
 
   /**
    * Get a list of summary data points
    *
-   * @return mixed; NULL or array with keys:
```

Comment:
I'd like to keep the @return documentation in these lines. The lint error here is ` Return type "mixed; NULL or array with keys:" must not contain spaces`. So I think you can fix this just by moving `NULL or array with keys:` to the next line.

Reply:
Fixed